src/api/classinfo.js

```diff
@@ -0,0 +1,31 @@
+import { instance } from '.';
+
+export const getClassinfo = async () => {
+  try {
+    const response = await instance.get(`/class-info`);
+    return response.data;
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export const updateClassinfo = async classInfo => {
+  const accessToken = localStorage.getItem('token');
+
+  if (!accessToken) {
+    console.error('No access token found, please log in.');
+    return;
+  }
```

Comment:
```suggestion
  const accessToken = localStorage.getItem('token');

  if (!accessToken) {
    console.error('No access token found, please log in.');
    return;
  }
```
이 부분 제가 한 번에 작성해서 없어도 돼요! 삭제해주세요~!